OpenSim/Simulation/Model/PathPoint.h

```diff
@@ -35,88 +35,76 @@ class PhysicalFrame;
 //=============================================================================
 //=============================================================================
 /**
- * A class implementing a path point.
- *
- * @author Peter Loan
- * @version 1.0
+ * A PathPoint that is stationary with respect to parent's PhysicalFrame
```

Comment:
This description ("A [type of] PathPoint that is stationary…") would make sense only for a class that *derives* from PathPoint. I suggest "A PathPoint" -> "A path point".